### ISSUE TITLE
Ability to add a heading to the Footer Menu

### ISSUE THREAD

**Issue by pikemurdy** [utkwdn/utkwds#401]
Currently there is no way to add a heading to the footer menu (second column of links) in the footer. Need to be able to add a header to that to provide context for users.

**Comment by reasterbrooknc** [comment]
@pikemurdy will review current functionality and regroup with internal UTK team to assess whether changes are needed to current footer.

**Comment by reasterbrooknc** [comment]
Per 1/23 meeting: Internal working happening on OCM side. @pikemurdy says this might get pushed but will follow up and let NC know.

**Comment by reasterbrooknc** [comment]
Is there finalized guidance here @pikemurdy?

**Comment by pikemurdy** [comment]
Yes: https://liveutk.sharepoint.com/sites/UniversityWebsiteResources/SitePages/WDS-Navigation-Guidelines.aspx

**Comment by pikemurdy** [comment]
Why is this one still blocked? @reasterbrooknc ? (before I do anything)

**Comment by reasterbrooknc** [comment]
> Okay, the thing I'm worried about now is, for folks who have already put a menu in the footer. (https://csw.utk.edu). Will their sites suddenly have a footer menu title?

Good question, yes, but @nc-mhenderson is working on an adjustment to address this. We'll let you know when he has that updated and what editors can expect.

**Comment by nc-mhenderson** [comment]
The Footer Links Menu now uses the **Display location** checkbox in Menu Settings. Previously, the menu name needed to be `Footer Links Menu`. The heading will display the menu name. If the menu is named `Footer Links Menu` the heading will not display.
